CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
 # Changelog
+## [2.0.5] - 2024-09-23
+### Breaking Changes
+- validate the tokenAddress for `enableSessionKey` on `SessionKeyValidator` SDK instance
```

Comment:
How is this a breaking change? Will it affect how existing users integrate our SDK?

Reply:
updated the change log